django/test/testcases.py

```diff
@@ -1255,15 +1255,14 @@ def captureOnCommitCallbacks(cls, *, using=DEFAULT_DB_ALIAS, execute=False):
         finally:
             callback_count = len(connections[using].run_on_commit)
```

Comment:
This line can be done once at the start of the loop, rather than before the loop and then at the end of each iteration.